### ISSUE TITLE
Consider removing .travis.yml and artifacts

### ISSUE THREAD

**Issue by cchris28** [dirac-institute/kbmod#211]
It appears left over from a bygone era. Do we use it anymore?

```
find . -name .git -prune -o -type f -print | xargs -r grep travis
./lib/pybind11/.travis.yml:    ${PYPY:+travis_wait 30} $PY_CMD -m pip install --user --upgrade pytest numpy scipy \
./lib/pybind11/README.md:[![Build Status](https://travis-ci.org/pybind/pybind11.svg?branch=master)](https://travis-ci.org/pybind/pybind11)
./src/kbmod.egg-info/SOURCES.txt:.travis.yml
```

**Comment by DinoBektesevic** [comment]
`/lib/pybind11/` would be a git submodule, they could be using it. The `.travis.yml` in the top dir can probably go - it's not up to date anymore and we found a solution using Baldur/Bendis. 

There are several other files like that - `clear.sh`, `run_tests.bash`, `hello.cpp` in `tests` dir and, in my opinion, the `kbmodpy.py` should go, but allegedly there are some notebooks out there that still use the overrides in it. This is the real problem with these files - not all of the analysis and code ended up in the repository, some still lives on outside it, and we're not sure how much and necessarily where exactly. Most of these I think could be cleared out safely though.

**Comment by jeremykubica** [comment]
Closed by #219
